blas/testblas/benchautogen/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Script for automatic code generation of the benchmark routines
+package main // import "gonum.org/v1/gonum/blas/testblas/benchautogen"
```

Comment:
Done. PTAL, this PR should be ready for merge unless some other issues are identified.